grpc-interface-support/src/python.py

```diff
@@ -128,6 +189,86 @@ def __copy_code_and_templates(
             template_dir, self.__package_directory_path, files_to_copy, variables
         )
 
+    def __create_stub_source(self) -> None:
```

Comment:
What kind of stub is it? Should be clear from the function name

Reply:
Renamed to "__create_service_stub_source". I know that "stub" is a bit generic for the "header class". Open for suggestions.